lib/smart_proxy_ansible/runner/ansible_runner.rb

```diff
@@ -10,6 +10,8 @@ class AnsibleRunner < ::Proxy::Dynflow::Runner::Parent
       include ::Proxy::Dynflow::Runner::ProcessManagerCommand
       attr_reader :execution_timeout_interval
 
+      ENVIRONMENT_WRAPPER = ENV['SMART_PROXY_ANSIBLE_ENVIRONMENT_WRAPPER'] || '/usr/libexec/ansible-runner-environment.sh'
```

Comment:
I'd prefer if the wrapper would not have a file extension of `.sh`, as it being shell is an implementation Detail.